inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/annotatorjs/resources/annotator-full.js

```diff
@@ -269,7 +269,10 @@
     node = root;
     for (_i = 0, _len = steps.length; _i < _len; _i++) {
       step = steps[_i];
-      _ref1 = step.split("["), name = _ref1[0], idx = _ref1[1];
+      step.split("[");
```

Comment:
This is a generated file (see comment at the beginning of file). Hence refactoring not necessary.